sdk-workflows/src/main/java/io/dapr/workflows/runtime/WorkflowRuntimeBuilder.java

```diff
@@ -16,10 +16,19 @@
 import com.microsoft.durabletask.DurableTaskGrpcWorkerBuilder;
 import io.dapr.utils.NetworkUtils;
 import io.dapr.workflows.Workflow;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.Set;
 
 public class WorkflowRuntimeBuilder {
   private static volatile WorkflowRuntime instance;
   private DurableTaskGrpcWorkerBuilder builder;
+  private static final Logger LOGGER = LoggerFactory.getLogger(WorkflowRuntimeBuilder.class);
+  private static volatile Set<String> activity_Set = new HashSet<String>(); 
```

Comment:
Why using static attributes in a builder class? Shouldn't this be instance bound?